{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -153,7 +153,7 @@ legacy_tox_ini = """
 
     [testenv]
     commands =
-        pytest --cov
+        pytest --cov --cov-report=xml
```

Comment:
Why `xml` format? I read something about `lcov` which is why I've used it https://github.com/paddyroddy/python-template/blob/762dc809b5d2b6ac201d6fcee304add0bec34792/pyproject.toml#L173, but that might have been something specific to `coveralls`